jubatus/core/framework/mixable_versioned_table.cpp

```diff
@@ -96,6 +96,10 @@ bool mixable_versioned_table::put_diff(const framework::diff_object& ptr) {
         core::common::exception::runtime_error("bad diff_object"));
   }
 
+  if (diff_obj->objs.empty()) {
+    return true;
+  }
+
   msgpack::object obj;
   obj.type = msgpack::type::ARRAY;
   obj.via.array.ptr = &diff_obj->objs[0];
```

Comment:
Notes for later reference:
This operation converts ``vector<msgpack::object> objs`` member of ``struct internal_diff`` into MessagePack array.  Actual values are not copied; only the address of the first element of the ``objs`` vector and its length is copied.

This issue does not affect any functionality, as ``obj.via.array.ptr`` (which is a invalid address) will never be accessed, because ``obj.via.array.size`` is 0.